smart-contracts/src/execute.rs

```diff
@@ -6,9 +6,10 @@ use crate::{
     state::{Config, FuelConfig, CONFIG, FUEL_CONFIG, LEADERBOARD},
 };
 use cosmwasm_std::{
-    coins, BankMsg, CosmosMsg, Decimal, DepsMut, Env, MessageInfo, Response, Uint128,
+    coins, ensure, BankMsg, CosmosMsg, Decimal, DepsMut, Env, MessageInfo, Response, Uint128,
 };
 use cw_utils::{must_pay, nonpayable, one_coin};
+use white_whale_std::pool_network::asset::is_ibc_token;
 
 pub fn execute_update_config(
```

Comment:
what if one of the fee recipient addresses needs to be changed? or the fee?

Reply:
The owner can update these

Reply:
how? you removed them from the update_config function

Reply:
I removed them from the `AddFuel` msg, not `UpdateFuelConfig`